src/node/rpc/frontend.h

```diff
@@ -354,6 +356,8 @@ namespace ccf
       client_signatures(client_sigs_),
       certs(certs_),
       callers(callers_),
+      pbft_requests(
```

Comment:
This solidifies pbft as a compile-time option, which isn't the direction we want to go in. Can we change it?

Reply:
This will be nullptr if we are running in Raft but I will check to see if there is a better way

Reply:
Replaced with a table lookup at runtime, let me know if you had something else in mind